kernel_hardening_checker/engine.py

```diff
@@ -112,6 +112,13 @@ def check(self) -> None:
         else:
             self.result = f'FAIL: "{self.state}"'
 
+        # handle checks, provided with list()
```

Comment:
How about moving it earlier and return?

Reply:
sure, i forgot about return(that is why i place this block of code close to end)
you're absolutely right